worlds/hk/__init__.py

```diff
@@ -131,6 +132,8 @@ class HKWeb(WebWorld):
         ["Ijwu"]
     )]
 
+    bug_report_page = "https://github.com/Ijwu/Archipelago.HollowKnight/issues/new/choose"
```

Comment:
```suggestion
    bug_report_page = "https://github.com/Ijwu/Archipelago.HollowKnight/issues/new?assignees=&labels=bug%2C+needs+investigation&template=bug_report.md&title="
```

Reply:
would it be better for it to go directly to bug report or to https://github.com/Ijwu/Archipelago.HollowKnight/issues so that people can see if the bug is already reported first?

Reply:
I'd rather they go directly to the bug report page, although you do have a good thought.